bin/http-server

```diff
@@ -92,10 +92,16 @@ else {
   listen(port);
 }
 
+/* jshint evil:true */
 function listen(port) {
+  var cache = argv.c;
+  try {
+    cache = eval(argv.c);
+  }
+  catch (e) {}
```

Comment:
Should we report to the user that their function failed?

Reply:
Added logging.